src/templates/blog.tsx

```diff
@@ -0,0 +1,79 @@
+import { graphql } from 'gatsby';
+import React from 'react';
+import Article from '../components/Article';
+import Layout from '../components/Layout';
+import { LearnPageContext, LearnPageData } from '../types';
```

Comment:
We should have our own types for blog.
The blog doesn't have a table of contents or description
just a nit it's not a blocking issue. Can you add a TODO so someone can update this later?

Reply:
The description of the blog comes from the excerpt actually. I'll update the types too